samples/robotframework-browser-library/tests/homepage-tests.robot

```diff
@@ -0,0 +1,19 @@
+*** Settings ***
+Library   Browser
+
+*** Test Cases ***
```

Comment:
These tests are rather odd. One goes to the playwright page, the others go to the testrail page. I'd suggest sticking to the testrail website and tests would be something like:
- Verify Header Links
- Verify Social Links
- Verify Trial Link (Invalid)

Reply:
Its done now @d-rede